README.md

```diff
@@ -52,21 +57,20 @@ Then run:
 
 Open the project in your favorite code editor `code .` for VS Code.
 
-
 ### Dependencies
-- Run npm install - to load necessary files from package.json
-- Export REACT_APP_API_KEY=ghp_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-> Except with your own key generated here https://github.com/settings/tokens
 
-- Then in your terminal type `npm start` and view app in your browser 
+- Run `npm install` - to install the necessary packages listed in the package.json file
+- Run `touch .env`
+- > Generate your GitHub token here https://github.com/settings/tokens
+- Run `echo REACT_APP_API_KEY=ghp_my_github_token > .env` make sure you replace `ghp_my_github_token` with your generated token
```

Comment:
This is great! 

Contributors should be able to get started with their contributions as soon as possible with these clear instructions! 👍